Tests/CorePromise/WhenTests.swift

```diff
@@ -218,7 +218,8 @@ class WhenTests: XCTestCase {
         let p2 = after(.milliseconds(100)).done { throw Error.straggler }
         let p3 = after(.milliseconds(200)).done { throw Error.straggler }
 
-        when(fulfilled: p1, p2, p3).catch { error -> Void in
+        let whenFulfilledP1P2P3: Promise<(Void, Void, Void)> = when(fulfilled: p1, p2, p3)
```

Comment:
There can be also `let whenFulfilledP1P2P3: Promise<Void> = when(fulfilled: p1, p2, p3)`.